Source/Yaml/_main_.env

```diff
@@ -171,8 +171,33 @@ _t4dSrcYamlUpdateLine(){
                 mv $_File.tmp $_File
             fi
         else
-            _t4dDebugLog $perror "Path not found >$_Path<"
-            return $_t4dErrorCode
+            _t4dDebugLog $pinfo "Creating $_Path"
+            local _Name="$(basename "$(echo $_Path | sed 's|\.|/|g' | sed 's|>|/|g')")"
+            local _Type="$(echo "${_Path}" | grep -Eo ".$_Name$" | cut -c-1 )"
+            local _RootPath="$(echo "${_Path}" | sed "s|.${_Name}$||g")"
+            
+
+            local _ParentLine="$(cat $_File | sed "s| |§SPACE§|g" | grep "^$_RootPath|")"
```

Comment:
you forgot to add the statement that we do not handle creating a path if rootPath does not exist and is not lvl0 dept

Reply:
I will call you afterwards for this point